R/tfrmt_layer.R

```diff
@@ -121,3 +121,81 @@ layer_tfrmt_arg.body_plan <- function(x, y, ...,  join_body_plans = TRUE){
 }
 
 
+
+#' Remap group values in a tfrmt
```

Comment:
Needs imported functions added to roxygen

Reply:
Fixed in c5ceec7